### ISSUE TITLE
Display Posts in Slider (Query Loop?)

### ISSUE THREAD

**Issue by iamtakashi** [WordPress/gutenberg#25502]
Similar to #25501, it could be interesting to think about what we could do to show posts in a slider. Like a carousel, we can think of many different variations of a slider, but as an example:

![Frame 62](https://user-images.githubusercontent.com/908665/93765035-2643b500-fc0c-11ea-9c18-c5b06a9a3515.jpg)

**Comment by hedgefield** [comment]
I agree this would be a really great addition to the List and Grid display options for the Query loop. I think https://github.com/WordPress/gutenberg/issues/25501 could be handled by this at the same time, you could probably set the slider to display one or several posts at the same time.

**Comment by frspp** [comment]
+1 this would be very good addition, used very much
